pyproject.toml

```
@@ -22,8 +22,8 @@ classifiers = [
 keywords = ['aiida', 'workflows']
 requires-python = '>=3.8'
 dependencies = [
-    'aiida_core[atomic_tools]~=2.1',
-    'aiida-pseudo~=0.8.0',
+    'aiida_core[atomic_tools]~=2.0,>=2.1.0',
+    'aiida-pseudo~=0.7.0',
```

```suggestion
    'aiida_core[atomic_tools]~=2.1',
    'aiida-pseudo~=0.7.0',
```
Should `aiida-pseudo` be downgraded?

---

Eh, not sure how that happened. Fixing now